src/training.jl

```diff
@@ -73,11 +73,11 @@ function train!(model,
             ) 
         end
 
-        # every so often, we upload a checkpoint to the cloud if applicable
+        # every so often, we save a checkpoint
         if epoch % freq_chckpt == 0
-            if !(logger isa Nothing)
-                log_artifact(logger, savepath; name="checkpoint-$(epoch)", type="BSON-file")
-            end
+            savepath = joinpath(savedir, "checkpoint_$(epoch).bson")
```

Comment:
@bischtob do we want this in general? if so, maybe Ill add in a separate PR and make freq_cheq = epochs in the Experiment tomls (so the default tomls is not to save many checkpoints)